### PR TITLE
Fix mobile swap container

### PR DESCRIPTION
**Pull request by biocom**
# Summary

- Makes the Swap container mobile responsive / more accessible
- Can now handle large numbers
- Addresses #1022 
- Removes top padding on mobile/responsive, to move up the container. This is fine, as long as we don't inherit the Uniswap new way of showing the main menu (which I would be against to).

tl;dr:
- More room for actual (input) numbers and legibility/readability of content.
- Does not sacrifice real estate (height) given padding removed.
- Ideally on a normal size phone, almost all of the swap container/form remains visible with user interaction.

**NEW -> CURRENT**
Screen Shot 2021-07-23 at 15 39 17Screen Shot 2021-07-23 at 17 00 05

**Other screens:**
Screen Shot 2021-07-23 at 15 39 05Screen Shot 2021-07-23 at 16 09 23Screen Shot 2021-07-23 at 16 08 49Screen Shot 2021-07-23 at 16 00 43Screen Shot 2021-07-23 at 15 39 57Screen Shot 2021-07-23 at 15 39 36Screen Shot 2021-07-23 at 16 45 58

Top padding mobile removed (NEW --> CURRENT):
Screen Shot 2021-07-23 at 16 50 20Screen Shot 2021-07-23 at 16 50 10

### CODE REVIEW

File: src/custom/components/CurrencyInputPanel/CurrencyInputPanelMod.tsx [hunk 1]
```diff
@@ -148,32 +148,6 @@ export const StyledBalanceMax = styled.button`
     margin-right: 0.5rem;
   `};
 `
-export const AuxInformationContainer = styled(Container)<{
```
Comment (biocom) [inline src/custom/components/CurrencyInputPanel/CurrencyInputPanelMod.tsx:151]: Wasn't part of the original UNI code (AFAIKT), thus moved to the `index` file to apply further mods.

File: src/custom/components/swap/SwapModalHeader/SwapModalHeaderMod.tsx [hunk 2]
```diff
@@ -27,7 +27,7 @@ import { INPUT_OUTPUT_EXPLANATION } from 'constants/index'
 import { computeSlippageAdjustedAmounts } from 'utils/prices'
 import { Field } from 'state/swap/actions'
 import { formatSmart } from 'utils/format'
-import { AuxInformationContainer } from 'components/CurrencyInputPanel/CurrencyInputPanelMod'
```
Comment (biocom) [inline src/custom/components/swap/SwapModalHeader/SwapModalHeaderMod.tsx:30]: Now part of the `index` file.

**Review by elena-zh — APPROVED** [review]
Great job!
I was able to catch this issue in the pr
image

However, this happens when enter an amount with >18 decimals into the field. This should be fixed in #1039 issue.

Thanks

**Comment by biocom** [comment]
@elena-zh Thank you for reviewing. I'm trying to replicate, but can't seem to (here I added extra text to try to replicate):
Screen Shot 2021-07-23 at 17 33 13

What steps should I follow to replicate?

**Comment by elena-zh** [comment]
Seems to be that it is a device-specific issue. 
I'm able to reproduce it in my iPhone 12 mini only. 

1. So I select From' token
2. Press on the 'Max' button
3. Edit a value: add extra decimals
4. Then I close the keyboard and see this broken text
https://user-images.githubusercontent.com/70885163/126806852-27fad9e9-eb81-4788-bf27-661fd06f5641.mp4

**Comment by biocom** [comment]
@elena-zh Nice bg music 🎵  - thx will try to replicate here with Xcode simulator.

**Review by alfetopito — APPROVED** [review]

**Pull request merged**